### PR TITLE
Allow any value text to be passed to the stepper so we can control the value

### PR DESCRIPTION
**Pull request by richardstevens**
#### What does this PR do? (please provide any background)
This allows us to send a value of our own into the input, this allows us to use the props to correctly increment and decrement but then display the result of this how we would like.

#### What tests does this PR have?
Usual tests and linting, i have updated the stepper test to include the setting of the valueText prop

#### How can this be tested?
This cant be tested until the new wizard repo is also updated

---

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

File: src/components/stepper/stepper.jsx [hunk 1]
```diff
@@ -49,6 +50,7 @@ module.exports = React.createClass({
   },
 
   render: function() {
+    var valueText = this.props.valueText ? this.props.valueText : this.props.value;
```
Comment (hxpaul) [inline src/components/stepper/stepper.jsx]: `var valueText = this.props.valueText || this.props.value`

**Comment by hxpaul** [comment]
Small recommended change but otherwise ok

**Comment by elliottcrush** [comment]
All good, works a treat.

**Comment by hxpaul** [comment]
review :one: all good now :+1:

**Pull request merged**